Admin.js

```diff
@@ -321,7 +321,7 @@ var Admin = {
                     counter = parseInt(matches[1], 10);
                 }
             });
-            Admin.collectionCounters[collection.attr('id')] = counter;
+            Admin.lolCounters[collection.attr('id')] = counter;
         });
     },
 
```

Comment:
- eslint: Missing JSDoc @returns for function.
- eslint: Missing JSDoc parameter description for 'subject'.
- eslint: Missing JSDoc parameter type for 'subject'.